### ISSUE TITLE
Feature Request: Show a name near the favourite snippet

### ISSUE THREAD

**Issue by chmouel** [SUPERCILEX/gnome-clipboard-history#68]
### Problem description

First thanks a lot for this great extension.

It is maybe not the right place since clipboard history is not a snippet manager, but I do keep a lot of quick snippet as favorite  in there. It would be great if we were able to set a name near the snippet (or instead of) to easy remember it.

### Potential solutions/workarounds

increase my consumption of Omega-3 fatty acid and Vitamine E to make my memory great again but that may not scale on the long term 
 🙃

**Comment by SUPERCILEX** [comment]
Haha, you might have to up your fish eating game unfortunately. :) There's a similar issue open and my answer is the same: https://github.com/SUPERCILEX/gnome-clipboard-history/issues/40.

For your use case, you can hack around it by including the title/name in the thing you copy, but of course that makes pasting a bit unpleasant.

**Comment by chmouel** [comment]
thanks, I i am not so sure tbh how to make this happen without being too awkward (and not sure i have the skills for it either)

**Comment by SUPERCILEX** [comment]
Yeah, I'm hoping someone will want to pick up the help wanted issues eventually.
